### ISSUE TITLE
Loss가 음수가 나온다.

### ISSUE THREAD

**Issue by GoX2Maker** [GoX2Maker/ConnectsLab_Speech#1]
![image](https://github.com/GoX2Maker/ConnectsLab_Speech/assets/131675046/65664305-e37f-415c-b352-2f8f94d627ce)

**Comment by GoX2Maker** [comment]
![image](https://github.com/GoX2Maker/ConnectsLab_Speech/assets/131675046/de260a5c-1d80-4acb-95e6-82cfe5e1beae)


float_out = out.float()  # ensure float32 for loss

# Pytorch CTC로 수정해서 아래처럼 해야함.
float_out = float_out.log_softmax(2) # ttps://pytorch.org/docs/stable/generated/torch.nn.CTCLoss.html#torch.nn.CTCLoss
loss = criterion(float_out, targets, output_sizes, target_sizes).to(device)
loss = loss / inputs.size(0)  # average the loss by minibatch
loss_value = loss.item()
